src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+**AppConfig**
+
+* Add support for importing/exporting feature flags
+
+2.0.78
+++++++
```

Comment:
Don't do 
```
2.0.78
++++++
```

Please rebase/merge the lates the dev branch

Reply:
done